### PR TITLE
Admin listing date expire

### PR DESCRIPTION
**Pull request by theRealRizeo**
https://github.com/Strategy11/BusinessDirectoryPlugin/issues/4975

Fixes setting listing to expired when date is changed by admin

### CODE REVIEW

**Review by garretlaxton — APPROVED** [review]
Changes work great. I tested this by changing the expiration date in the backend, as well as imports. 
Screenshot 2021-10-14 at 15-24-33 Directory ‹ Business Directory — WordPress

**Review by truongwp — APPROVED** [review]
@theRealRizeo Your code looks good!

File: includes/models/class-listing.php [hunk 1]
```diff
@@ -734,7 +734,9 @@ public function update_plan( $plan = null, $args = array() ) {
 
         if ( is_null( $row['expiration_date'] ) || empty( $row['expiration_date'] ) ) {
             unset( $row['expiration_date'] );
-        }
+        } elseif ( strtotime( $row['expiration_date'] ) < current_time( 'timestamp' ) ) {
```
Comment (stephywells) [inline includes/models/class-listing.php:737]: This looks great now! Can you please check tabs vs spaces on all changed lines? Once that is fixed, we can merge this.

If you're not sure how to find them easily, take a look at the diff.
https://github.com/Strategy11/business-directory-plugin/pull/54/files

**Review by stephywells — CHANGES_REQUESTED** [review]

File: includes/models/class-listing.php (hunk 1 above)
Comment (theRealRizeo) [reply includes/models/class-listing.php]: Updated :)

**Review by stephywells — APPROVED** [review]
Awesome! Looks perfect now!

**Pull request merged**